<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Vnos kurilne naprave</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Vrsta</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Namen</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Lokacija</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Znamka</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Tip</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Moč</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Letnik</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Zalaganje</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Opombe</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vnos naprave za naslednjega uporabnika:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUporabnik">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Deli kurilne naprave</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Vrsta odvadoa</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Material</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_11"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Premer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Višina</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_13"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Iztočnica</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_14"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Opombe</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
